src/services/pie-stat-getter.js

```diff
@@ -3,17 +3,15 @@ import P from 'bluebird';
 import moment from 'moment';
 import { Schemas } from 'forest-express';
 import { isVersionLessThan4 } from '../utils/orm';
-import BaseStatGetter from './base-stat-getter';
 import { isMSSQL } from '../utils/database';
+import ConditionsParser from './conditions-parser';
 
 // NOTICE: These aliases are not camelcased to prevent issues with Sequelize.
 const ALIAS_GROUP_BY = 'forest_alias_groupby';
 const ALIAS_AGGREGATE = 'forest_alias_aggregate';
 
-function PieStatGetter(model, params, opts) {
-  BaseStatGetter.call(this, model, params, opts);
-
-  const needsDateOnlyFormating = isVersionLessThan4(opts.sequelize);
+function PieStatGetter(model, params, options) {
```

Comment:
You should move out the `opts` renaming out of the PR to ease the work of your reviewer ;)